docs/cisco.ios.ios_config_module.rst

```diff
@@ -507,7 +507,7 @@ Notes
 
 .. note::
    - Tested against IOS 15.6
-   - Abbreviated commands are NOT idempotent, see L (Network FAQ,../network/user_guide/faq.html#why-do-the-config-modules-always-return-changed-true-with-abbreviated-commands).
+   - Abbreviated commands are NOT idempotent, see `Network FAQ <../network/user_guide/faq.html#why-do-the-config-modules-always-return-changed-true-with-abbreviated-commands>`_.
```

Comment:
You need to change this in the actual python file, not the generated rst file please.

Reply:
Do you have a recommended setup that is different than what is showing up on the NXOS link that is working? What python file? It should be a page view shouldn't it since it is related to all networking in general?

Reply:
Ah sorry, thought you were one of the regulars here, your name seemed so familiar!  Give me a bit to gather the relevant info and post it here. And thanks for lending a hand!